src/controller/handler/connections.ts

```diff
@@ -54,10 +54,10 @@ export class Connections implements IAgentResponseHandler {
         const delay = (ms: number) => {
             return new Promise(resolve => setTimeout(resolve, ms));
         };
-        const readPermission = (governanceKey: string, cacheKey: string) => {
+        const readPermission = (governanceKey: string, _cacheKey: string) => {
```

Comment:
Was cacheKey prefixed with _ because there is a similar named variable below?  I don't like underscores as I find it harder to read.  I would rather see one of the variables named differently.  What's your thoughts?

Reply:
Yeah, tslint was complaining about it being a shadowed variable. I do agree, thought it might be better to rename one of the other variables, but also didn't want to accidentally remove some semantic meaning. How about naming the other one `templatedCacheKey` or something like that?